File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
    let lux = Infinity;
    let luy = Infinity;
    let rdx = -Infinity;
    let rdy = -Infinity;

    for (let i = 0; i < wallpaper.length; i++) {
        for (let j = 0; j < wallpaper[i].length; j++) {
            if (wallpaper[i][j] === '#') { 
                lux = Math.min(lux, i);
                rdx = Math.max(rdx, i + 1);
                luy = Math.min(luy, j);
                rdy = Math.max(rdy, j + 1);
            }
        }
    }

    return [lux, luy, rdx, rdy]; 
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    lux = float('inf')
    luy = float('inf')
    rdx = 0
    rdy = 0
    
    for x,row in enumerate(wallpaper):
        for y in range(len(row)):
            if wallpaper[x][y] == "#":
                lux = min(lux,x)
                luy = min(luy,y)
                rdx = max(rdx,x+1)
                rdy = max(rdy,y+1)
            
    return [lux,luy,rdx,rdy]